repos/system_upgrade/common/actors/targetuserspacecreator/tests/unit_test_targetuserspacecreator.py

```diff
@@ -1,5 +1,7 @@
 import os
```

Comment:
The division without future message from pylint is kind of a false positive in this case (it won't affect the division operator used with pathlib) but it's easy enough to fix:

```suggestion
from __future__ import division

import os
```

Reply:
Thanks! Good thing I did not rewrite the whole thing using os.join-s as I intended originally...